frontend/js/src/settings/brainzplayer/BrainzPlayerSettings.tsx

```diff
@@ -96,20 +96,14 @@ function BrainzPlayerSettings() {
       <Helmet>
         <title>BrainzPlayer Settings</title>
       </Helmet>
-      <h2 className="page-title">BrainzPlayer Settings</h2>
-      <h4 className="page-title">Music Players</h4>
+      <h2 className="page-title">BrainzPlayer settings</h2>
       <p>
-        You can choose which music services to play music from on listenBrainz.
-        To sign in and authorize your account please go to the{" "}
-        <Link to="/settings/music-services/details/">
-          &quot;connect services&quot; page
-        </Link>
-        . <br />
-        If you don&apos;t have a subscription to a music service listed below,
-        Youtube will be used by default as it does not require an account.
-        However the search results from Youtube are often inaccurate, and for a
-        better listening experience we recommend one of the other services (if
-        you have one available).
+        Choose which music services to use for playback in ListenBrainz.
+      </p>
+      
+      <p?
```

Comment:
This is the only actual issue, malformed HTML tag.
If it's of any use in the future, you can see this issue marked in the "files" tab of the pull request on github, it's a good way to catch errors even if you can't test it yourself locally: https://github.com/metabrainz/listenbrainz-server/pull/2856/files
![image](https://github.com/metabrainz/listenbrainz-server/assets/6179856/053ee283-ceb7-4dca-9542-1bf3122635cc)

```suggestion
      <p>
```

Reply:
Fixed it